docs/compass/02-01-compass.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Basic architecture
+type: Architecture
+---
+
+Compass is a central place which stores applications and runtimes configurations, and then propagates that information accordingly. It also plays a crucial role in establishing a trusted connection between applications and runtimes. The basic workflow looks as follows:
+
+![Basic architecture](./assets/architecture.svg)
+
+1. Administrator adds runtimes and applications, and configures them using Compass GraphQL API.
+2. Agent, a component that is integrated in every Kyma runtime, continuously fetches the actual configuration from Compass. It also sends information about runtime health checks to Compass.
```

Comment:
health checks are not yet implemented, should we mention them in docu?

Reply:
Agreed with PK that we'll use the future tense here.